local/compose/attach.go

```diff
@@ -114,7 +114,7 @@ func (s *composeService) getContainerStreams(ctx context.Context, container moby
 			Stdin:  true,
 			Stdout: true,
 			Stderr: true,
-			Logs:   true,
+			Logs:   false,
```

Comment:
Looking back at the history of this (3714ab7d41af64f456e110e572681d570a598c1d), this was set to true because we were loosing some logs in `run` (with a race probably). Is this not needed anymore ?

Reply:
yes indeed.
according to API docs `Logs: Replay previous logs from the container` is the main reason we get logs from previous runs as described in #1289
this option is usefull to attach to a container that's already running, but here we attach before we call `start`, so not needed